packages/modals/src/elements/TooltipModal/Title.tsx

```diff
@@ -5,15 +5,27 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import React, { HTMLAttributes, forwardRef } from 'react';
+import React, { HTMLAttributes, forwardRef, useEffect } from 'react';
 import PropTypes from 'prop-types';
 import { useTooltipModalContext } from '../../utils/useTooltipModalContext';
 import { StyledTooltipModalTitle } from '../../styled';
 import { ITooltipModalTitleProps } from '../../types';
 
 const TitleComponent = forwardRef<HTMLDivElement, ITooltipModalTitleProps>(
   ({ children, tag, ...other }, ref) => {
-    const { getTitleProps } = useTooltipModalContext();
+    const { getTitleProps, isTitlePresent, setIsTitlePresent } = useTooltipModalContext();
```

Comment:
[nit] garden tends to use `has...` notation for component existence. i.e. `hasTitle` & `setHasTitle`.

Reply:
Sounds good. I saw a couple different phrasings. :)